.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
+
+executors:
+  node:
+    docker:
+      - image: node:12
+        user: node
+  gcloud:
+    docker:
+      - image: google/cloud-sdk:alpine
+
+commands:
+  deploy-env:
+    description: "Deploy to a specific environment"
+    parameters:
+      sa_key_var:
+        type: string
+        default: "SA_KEY_JSON"
```

Comment:
Why have a default here if it's not dev?

Reply:
In preparation for deployment to other environments, if we are not going to have other envs then this can be removed